tests/integration/welcomeMessage.test.ts

```diff
@@ -0,0 +1,138 @@
+import "@/testUtils/unit/mockDb";
+
+import { App } from "@slack/bolt";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import { afterEach, beforeAll, describe, expect, it, vi } from "vitest";
+
+import { iceBreakerThreads, presentIdeas, users } from "@/db/schema";
+import { getIceBreakerThreadLink } from "@/services/slack/getIceBreakerThreadLink";
+import { timeout } from "@/testUtils/constants";
+import { deleteLastDm } from "@/testUtils/integration/deleteLastDm";
+import { deleteLastRandomChannelPost } from "@/testUtils/integration/deleteLastRandomPost";
+import { sendCronEvent } from "@/testUtils/integration/sendCronEvent";
+import { sendScheduleEvent } from "@/testUtils/integration/sendScheduleEvent";
+import { waitForDm } from "@/testUtils/integration/waitForDm";
+import { testDb, waitForIceBreakerThreads } from "@/testUtils/testDb";
+
+dayjs.extend(utc);
+
+const constants = vi.hoisted(() => ({
+  teamId: import.meta.env.VITE_SLACK_TEAM_ID,
+  userId: "U1",
+  otherUserId: import.meta.env.VITE_SLACK_BOT_USER_ID,
+  presentIdeas: [
+    {
+      userId: import.meta.env.VITE_SLACK_BOT_USER_ID,
+      presentIdea: "Test idea",
+    },
+    {
+      userId: import.meta.env.VITE_SLACK_BOT_USER_ID,
+      presentIdea: "Test idea 2\nHello\n\n- Idea1\n- Idea2\n- Idea3",
+    },
+  ],
+}));
+
+vi.mock("sst/node/config", () => ({
+  Config: {
+    RANDOM_SLACK_CHANNEL_ID: import.meta.env.VITE_RANDOM_SLACK_CHANNEL_ID,
+  },
+}));
+
+vi.mock("@/services/slack/createSlackApp", async () => ({
+  createSlackApp: vi.fn().mockReturnValue(
+    new App({
+      signingSecret: import.meta.env.VITE_SLACK_SIGNING_SECRET,
+      token: import.meta.env.VITE_SLACK_BOT_TOKEN,
+    }),
+  ),
+}));
+
+describe("Create birthday squad", () => {
```

Comment:
This should be Send welcome message, also the file name should be sendSquadWelcomeMessage after the lambda, just like most of the other tests.

Reply:
Renamed it, also I used to publish the createBirthdaySquad but it's better to publish the sendSquadWelcomeMessage directly.